common/changes/@uifabric/dashboard/AddtooltipForIcons_2019-03-19-09-11.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/dashboard",
+      "comment": "\"have added the tooltip for the refresh icon and close icon in the detail panel with pivot\"",
+      "type": "patch"
```

```suggestion
      "type": "minor"
```

Unsure if Dashboard package follows the same SemVer rules as OUFR proper (I think it does). If so this should be of type `minor` since you're adding to the API.